grpc/binlog/v1alpha/binarylog.proto

```diff
@@ -97,12 +97,13 @@ message MetadataEntry {
 message Peer {
   enum PeerType {
     UNKNOWN_PEERTYPE = 0;
-    PEER_IPV4 = 1;  // peer is struct sockaddr_in
-    PEER_IPV6 = 2;  // peer is struct sockaddr_in6
-    PEER_UNIX = 3;  // peer is struct sockaddr_un
+    PEER_IPV4 = 1;  // peer is 4 byte ipv4 address
```

Comment:
That won't be human readable

Reply:
I'm all for making things more readable. What do you think about using the [canonical ipv6](https://tools.ietf.org/html/rfc5952#section-4) address format?